content/docs/tasks/telemetry/distributed-tracing/jaeger/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Jaeger
+description: How to configure the proxies to send tracing requests to Jaeger.
+weight: 10
+keywords: [telemetry,tracing,jaeger]
```

Comment:
The keywords determine the topics shown at the bottom of the page under "See also:" I would include things like span, port forwarding, and Zipkin at the very least.